addons/core/translations/resources/en-us.yaml

```diff
@@ -384,8 +384,8 @@ session-recording:
     messages:
       none:
         title: Channel Playback
-      missing:
-        title: We can't play back this channel because the file is missing
+      error:
+        title: Playback error
         description: This likely means the connected storage bucket is misconfigured or the file was deleted.
```

Comment:
Shouldn't this description change as well since a lot of times it won't be either of those cases?

Reply:
good point, ill reach out to design on this

Reply:
updated!